apps/web/src/pages/quick-start/components/demo-flow/DigestDemoFlow.tsx

```diff
@@ -60,14 +64,26 @@ const nodes: Node[] = [
     type: 'emailNode',
     data: {
       label: 'Email',
+      ActionItem: DotsHorizontal,
     },
-    position: { x: NODE_INITIAL_X, y: NODE_INITIAL_Y + NODE_DIST + NODE_DIST },
+    position: { x: NODE_INITIAL_X, y: NODE_INITIAL_Y + NODE_DIST * 2 },
+  },
+  {
+    id: '4',
+    type: 'addNode',
+    data: {
+      label: 'Add Node',
+      ActionItem: DotsHorizontal,
+    },
+    position: { x: NODE_INITIAL_X, y: NODE_INITIAL_Y + NODE_DIST * 2 + 100 },
```

Comment:
I feel like it's better not to add it because it anticipates that you might be able to click it... also on the Digest Playground we don't have it...

Reply:
Tend to agree (i thought the same for the three dots on the preview page [ref](https://www.figma.com/file/hlsUxjq2kxUU7aVBf2QyGu?node-id=409:41745#384032833)) I added it only due to the reason it exists in the design :)

Reply:
@LetItRock You think we should remove it?

Reply:
@djabarovgeorge yes